django/db/backends/oracle/base.py

```diff
@@ -387,8 +388,7 @@ def __init__(self, connection):
         self.cursor = connection.cursor()
         # Necessary to retrieve decimal values without rounding error.
         self.cursor.numbersAsStrings = True
-        # Default arraysize of 1 is highly sub-optimal.
-        self.cursor.arraysize = 100
+        self.cursor.arraysize = constants.GET_ITERATOR_CHUNK_SIZE
```

Comment:
I figured it might make sense to access other SQL constants in this module in the future and that it would avoid changing the import. I've no strong opinion on it though.